src/NovelRT.Interop/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
-set(INTEROP_SOURCES
+set(SOURCES
   NrtDebugService.cpp
   NrtInteropErrorHandling.cpp
   NrtLoggingService.cpp
   NrtNovelRunner.cpp
   NrtTransform.cpp
-  
+
```

Comment:
Uhhhh, what changed here?

Reply:
Removal of whitespace

Reply:
Whitespace was removed automatically by the formatter.

This was a line that was `  \n` (two spaces and a newline), it is now just `\n` (just a newline)